### ISSUE TITLE
Change event fired on form-checkbox mounted hook

### ISSUE THREAD

**Issue by javiercbk** [bootstrap-vue/bootstrap-vue#134]
The following element fires the change event upon creation

```html
<b-form-checkbox @change="onChange($event)" :checked="visible"></b-form-checkbox>
```

This behavior seems to be intentional as state in `form-checkbox.vue`.

```js
mounted() {
  this.change(this.localChecked);
},
```

The problem is that whenever I register the `change` event listener on a regular checkbox, it does not fire the change event right away and it seems that this component should not either.

Is there a good reason for this behavior? am I missing something here?

**Comment by pi0** [comment]
@javiercbk Sorry for reaching you out late. actually there was implementation problems with vue as it was forcing `model` prop and `input` event. This was fixed in vue [v2.0.2 initial](https://github.com/vuejs/vue/releases/tag/v2.2.0) release. we will have many checkbox improvements in next alpha release ;)

**Comment by javiercbk** [comment]
Nice. Thanks
